src/test/ui/fn/fn-recover-return-sign.fixed

```diff
@@ -0,0 +1,28 @@
+// run-rustfix
+#![allow(unused)]
+fn a() -> usize { 0 }
+//~^ ERROR return types are denoted using `->`
+
+fn b()-> usize { 0 }
```

Comment:
How come rustfix does not add a space here?

Reply:
Rustfmt doesn't run on the test suite (and shouldn't, because sometimes the tests are for the parser, and the parser should be whitespace-independent).

Reply:
Isn't this the output of rustfix? Shouldn't rustfix code output be sightly formatted (at least not as good as rustfmt)?

Reply:
Oh! I missed that this is a `.fixed` file. This happens because the original doesn't have a space before the `:`:

```
fn b(): usize { 0 }
//~^ ERROR return types are denoted using `->`
```
Ideally the suggestion would add its own space, yeah.